src/exml_query.erl

```diff
@@ -44,7 +52,8 @@ path(Element, Path) ->
 path(#xmlel{} = Element, [], _) ->
     Element;
 path(#xmlel{} = Element, [{element, Name} | Rest], Default) ->
-    Child = subelement(Element, Name), % may return undefined
+    Child = subelement(Element,
+                       Name), % may return undefined
```

Comment:
Why the line break? Sometimes the formatter compacts stuff onto one line, but in this case it inserted a line break due to no apparent reason.

Reply:
This is odd, indeed.
Please report this as a bug on the rebar3_format repo.
On the other hand, have you tried with different values for `paper` and/or `ribbon` in your configuration? Maybe you have a too narrow `paper` configuration.

Reply:
We have the default vaules `paper => 100` and `ribbon => 90`